src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -55,33 +55,44 @@ public JwtTokenReader(TokenValidationParameters tokenValidationParameters, strin
         }
 
         /// <summary>
-        ///     Validates if the token is considered valid
+        ///     Validates if the token is considered valid.
```

Comment:
I'd say "Verify if the token is valid" instead :)

Reply:
Done that.